packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -91,9 +91,17 @@ The context object allows [trusted web applications](#trusted-application) such
 | ----------- | ----------------------------------------------------------------------------- | -------- | -------- | ------ | -------- | --------- | --------- |
 | deviceToken | A globally unique ID identifying the user's client device or user agent | String   | TRUE     | FALSE  | FALSE    |           | 32        |
 
-> **Note:** You must always pass the same `deviceToken` for a user's device with every authentication request for **per-device** or **per-session** Sign-On Policy Factor challenges. If the `deviceToken` is absent or does not match the previous `deviceToken`, the user is challenged every-time instead of **per-device** or **per-session**.
+> **Note:** You must always pass the same `deviceToken` for a user's device with every authentication request for **per-device** or **per-session** Sign-On Policy Factor challenges. If the `deviceToken` is absent or does not match the previous `deviceToken`, the user is challenged every-time instead of **per-device** or **per-session**. Similarly you must always pass the same `deviceToken` for user's device with every authentication request for **new device security behavior detection**. If the `deviceToken` is absent or does not match a recent `deviceToken` for the user, the request will be considered to be from a new device. For more information about this feature, see [Security Behavior Detection](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm).
 
-It is recommended that you generate a UUID or GUID for each client and persist the `deviceToken` as a persistent cookie or HTML5 localStorage item scoped to your web application's origin.
+##### Device Token Best Practices
+
+Use the following recommendations as guidelines for generating and storing a `deviceToken` for both web and native applications.
+
+**Web Apps**<br>
+Okta recommends that you generate a UUID or GUID for each client and persist the `deviceToken` using a secure, HTTP-only cookie or HTML5 localStorage scoped to the customer's domain as the default implementation. See [Cookie flags that matter](https://odino.org/security-hardening-http-cookies/#cookie-flags-that-matter) for more best practices on hardening HTTP cookies.
```

Comment:
should localStorage be in tics? `localStorage`

Reply:
i don't know. I copied this from else where. I think this is a javascript term.

Reply:
It's a common term for a form of browser memory, right? Don't think it needs to be code formatted.